photon-client/src/components/app/photon-camera-stream.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+import { computed } from "vue";
+import { useCameraSettingsStore } from "@/stores/settings/CameraSettingsStore";
+import { useStateStore } from "@/stores/StateStore";
+import loadingImage from "@/assets/images/loading.svg";
+import type { StyleValue } from "vue/types/jsx";
+
+const props = defineProps<{
+  streamType: "Raw" | "Processed",
+  id?: string
+}>();
+
+const src = computed<string>(() => {
+  if(!useStateStore().backendConnected) {
+    return loadingImage;
+  }
+
+  const port =  useCameraSettingsStore().currentCameraSettings.stream[props.streamType === "Raw" ? "inputPort" : "outputPort"];
+
+  return `http://${location.hostname}:${port}/stream.mjpg`;
```

Comment:
we got rid of the seed? why?

Reply:
I did testing both with and without the random seed. On my browser (Chromium 114) I noticed no difference both with and without random seed generation. You can also just see this when the image is opened on its own in its own tab. Anything online dealing with wjpg streams doesn't recommend that seed generation thing. If we notice loading time is an issue for the camera streams, we can eagerly load them but I'm not sure how that would work with the backend not connected.

Reply:
I'd err on the side of caution with these kinds of things. In the 2022 pre-season, we faced a lot of random issues with camera stream inconsistency so I'd either test it more on other browsers/machines/networking conditions or keep it the same way.

Reply:
Worst case we can add it back but there was also a lot of blocking JS for fast image loading on the old UI so I think we should use this year's beta to try it out and then we can add it back if we need to. Again, I did test it side by side for like an hour on OSX and Windows and noticed no difference.

Reply:
I'm loathe to toach any networking things. Since nobody has an FMS to test with, we're stuck pushing broken code to beta teams at offseasons. Or like the time when my rio finder bricked everyone's photon right following autonomous only on real fms at competitions.

But maybe there was a browser bug to blame for this all along. Regardless, it costs us nothing to keep.